client/packages/system/src/Report/components/ReportArgumentsModal.tsx

```diff
@@ -68,11 +68,10 @@ export const ReportArgumentsModal: FC<ReportArgumentsModalProps> = ({
           }}
         />
       }
+      contentProps={{ sx: { margin: '0 auto' } }}
```

Comment:
some styling is a little off in full screen mode, i need to review this more fully